tensorflow-core/tensorflow-core-api/src/gen/java/org/tensorflow/internal/c_api/global/tensorflow.java

```diff
@@ -3150,7 +3150,7 @@ public static native void TF_RegisterFilesystemPlugin(
 // TF_InitKernel to do op/kernel registration.
 // Plugin should implement TF_InitKernel to register kernels. This function
 // should register all kernels in a plugin.
-public static native void TF_InitKernel();
+
```

Comment:
Note that I know where this change comes from :) I'm skipping this C API endpoint that was causing trouble on TF2.5 but I forgot to commit the updated generated bindings so let's keep it here